ome/figure
# Luts

This adds LUTs support to OMERO.figure.

To test:
 - click on channel color chooser drop-down and select "Lookup Tables".
 - This will open a dialog showing available LUTs
 - Click a LUT to preview it, then OK
 - Channel button AND slider should have LUT background.
 - This should also work to set the channel LUT when multiple images are selected, even if they have different LUTs/colors for that channel before setting LUT.
 - Export to PDF/TIFF should export images with LUTs applied as in the figure.

<img width="1012" alt="screen shot 2016-08-03 at 23 16 22" src="https://cloud.githubusercontent.com/assets/900055/17383908/6ec37a3c-59d0-11e6-8ea9-9e4a104d9f91.png">
